### ISSUE TITLE
feat: click range and hint

### ISSUE THREAD

**Issue by Omega0x013** [Omega0x013/the-pond#17]
Currently, the player can click on any lily pad onscreen. This means the range of a mouse or mobile user is greater than that of a keyboard user. A max range on click identical to that of keyboard should be enforced.

Whenever any method can't find a lily pad in the desired search zone, it should flash a grey semitransparent dashed circle showing the scan radius for jumps.
